### PR TITLE
update documentation description

### PR DESCRIPTION
**Pull request by cyberliem**

### CODE REVIEW

**Comment by cyberliem** [comment]
[Improve Readme for SDK](https://trello.com/c/oYlB511e/30-improve-readme-for-sdk)

File: README.md [hunk 1]
```diff
@@ -61,26 +88,99 @@ const dpl = new Deployer(window.web3.currentProvider);
...
+The following example set the base rate for KNC token.
+
+```js
+const reserve = new Reserve(provider, addresses);
+const operatorAccount = web3.eth.accounts.privateKeyToAccount('operatorAccountPrivateKey');
+const KNCTokenAddress = "0x095c48fbaa566917474c48f745e7a430ffe7bc27"
```
Comment (favadi) [inline README.md]: please add ';' at the end of line.

Comment (favadi) [reply README.md]: please review the same errors everywhere.

File: README.md [hunk 2]
```diff
@@ -61,26 +88,99 @@ const dpl = new Deployer(window.web3.currentProvider);
...
+  const reserve = new Reserve(provider, addresses);
+  const adminAccount = web3.eth.accounts.privateKeyToAccount('adminAccountPrivateKey');
+  const operatorAccount = web3.eth.accounts.privateKeyToAccount('operatorAccountPrivateKey');
+  const alerterAccount = web3.eth.accounts.privateKeyToAccount('alerterAccountPrivateKey');
+  const receiverAddress = '0x69E3D8B2AE1613bEe2De17C5101E58CDae8a59D4' 
+  const KNCTokenAddress = "0x095c48fbaa566917474c48f745e7a430ffe7bc27"
+
+  (async () => {
+    // stop trade. 
+    await reserveContract.disableTrade(alerterAccount)
```
Comment (favadi) [inline README.md]: it should be reserve.reserveContract?

Comment (cyberliem) [reply README.md]: no it's a wrapper method as [here](https://doc.esdoc.org/github.com/KyberNetwork/reserve-sdk.js/class/src/reserve.js~Reserve.html#instance-method-disableTrade)

**Pull request merged**